lib/src/Pos/Binary/Communication.hs

```diff
@@ -81,6 +84,50 @@ serializeMsgSerializedBlock :: MsgSerializedBlock -> BS.ByteString
 serializeMsgSerializedBlock (MsgSerializedBlock b) = "\x82\x0" <> unSerialized b
 serializeMsgSerializedBlock (MsgNoSerializedBlock t) = serialize' (MsgNoBlock t)
 
+serializeMsgStreamBlock :: MsgSerializedBlock -> LBS.ByteString
```

Comment:
Needs a comment similar to the one for `serializeMsgSerializedBlock`

Reply:
Will address.